docs/reference-guides/block-api/block-supports.md

```diff
@@ -88,11 +88,12 @@ supports: {
     -   `background`: type `boolean`, default value `true`
     -   `__experimentalDuotone`: type `string`, default value undefined
     -   `gradients`: type `boolean`, default value `false`
+    -   `link`: type `boolean`, default value `true`
     -   `text`: type `boolean`, default value `true`
 
 This value signals that a block supports some of the properties related to color. When it does, the block editor will show UI controls for the user to set their values.
 
-Note that the `text` and `background` keys have a default value of `true`, so if the `color` property is present they'll also be considered enabled:
+Note that the `background`, `link`, and `text` keys have a default value of `true`, so if the `color` property is present they'll also be considered enabled:
```

Comment:
For `link`, this is the other way around: it's `false` by default and doesn't get enabled by `color`.

Reply:
Thanks @nosolosw, I thought I had read in the PR they would be the same.
I updated the docs in https://github.com/WordPress/gutenberg/pull/32936/commits/f11646d00932f9e1bddd7ef70645428575a737b4 to have `false` be the correct default value